fast64_internal/f3d/f3d_material.py

```diff
@@ -3820,7 +4084,15 @@ def mat_register():
         items=enumF3D,
         default="F3D",
     )
-    bpy.types.Scene.isHWv1 = bpy.props.BoolProperty(name="Is Hardware v1?")
+    bpy.types.Scene.isHWv1 = bpy.props.BoolProperty(
```

Comment:
There's only a couple SGI Indy N64 devkits owned by collectors which have V1 chips. (Actually not sure if the chips in them are V1 or not.) No retail consoles are V1 and even good emulators will not support V1 display lists. The fast64 support for V1 appears to be because it was in the GBI when that was ported to Python. I'm happy to remove it from the codebase if you are okay with that.

Reply:
I'd like to hear more fast64 opinions on this but yeah I think we should make it deprecated